.github/workflows/android-release.yml

```
@@ -22,8 +22,33 @@ jobs:
 
       - name: Build Debug APK
         run: ./gradlew assembleDebug
+        
+      - name: Get short commit hash
+        id: commit
+        run: echo "sha=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+        
+      - name: Get current date
+        id: date
+        run: echo "date=$(date +'%Y-%m-%d')" >> $GITHUB_OUTPUT
 
-      - name: Upload Debug APK
+      - name: Create GitHub Release
+        id: create_release
+        uses: softprops/action-gh-release@v1
```

```suggestion
        uses: softprops/action-gh-release@v2
```

We might need `v2` here for the `make_latest` flag to be available. https://github.com/softprops/action-gh-release/pull/304#issuecomment-1986457346